src/applications/vaos/services/location/index.js

```diff
@@ -169,13 +149,35 @@ export async function getLocationsByTypeOfCareAndSiteIds({
         facilityIds: uniqueIds,
       });
 
+      const requestSupportedFacilityIds =
```

Comment:
We could probably refactor this service to do all types of care at once, and leave the filtering to Redux/components, since we're passing back all facilities now. That would let us avoid having to put the facilities array in a object with type of care id keys.

Reply:
Yes true, good call.